include/deal.II/lac/la_parallel_vector.h

```diff
@@ -346,6 +346,20 @@ namespace LinearAlgebra
              const IndexSet &ghost_indices,
              const MPI_Comm  communicator);
 
+      /**
+       * Initialize vector with @p local_size locally-owned and @p ghost_size
+       * ghost degrees of freedoms.
+       *
+       * @note The created underlying partitioner is set up so that the
+       *   number of locally-owned and ghost degrees of freedom match the given
+       *   parameters. However, an interpretation of the ghost indices is not
+       *   possible.
+       */
+      void
+      reinit(const size_type local_size,
+             const size_type ghost_size,
+             const MPI_Comm  comm);
```

Comment:
Did you forget to include the definition of this function? I cannot find it below.

Reply:
The implementation is on include/deal.II/lac/la_parallel_vector.templates.h.